src/lib.rs

```diff
@@ -399,11 +418,16 @@ fn pan_orbit_camera(
                     true => -1.0,
                     false => 1.0,
                 };
-                let multiplier = match ev.unit {
-                    MouseScrollUnit::Line => 1.0,
-                    MouseScrollUnit::Pixel => 0.005,
+                let delta_scroll = ev.y * direction * pan_orbit.zoom_sensitivity;
+                match ev.unit {
+                    MouseScrollUnit::Line => {
+                        scroll_line += delta_scroll;
+                    }
+                    MouseScrollUnit::Pixel => {
+                        scroll_pixel += delta_scroll * 0.05;
```

Comment:
What's the reason for changing the multiplier from 0.005 to 0.05?

Reply:
Typed wrong